### DISCUSSION TITLE
Is trial time taken into account when performing the optimization?

### DISCUSSION THREAD

**Discussion by jmribeiro** [optuna/optuna#4447] [Q&A]
Let's say you have two trials T1 and T2 (each with their parameters).
Both achieve the same objective value, but T1 took 1h to run and T2 20 minutes.

Is this time used to assess that T1's parameters are worse than T2?
Or in terms of objective, are they considered the same?

**Comment by nzw0301** [comment]
I think Optuna uses only objective value; they are treated equally. One possible approach is to use multi-objective function, but it might not be easy to select the best hyper-params.
